extensions/qonto/src/components/bank-accounts-list.tsx

```diff
@@ -0,0 +1,158 @@
+import { useMemo, useState } from "react";
+import { Icon, List, ActionPanel, Action, Color, useNavigation, Detail } from "@raycast/api";
+import { currency } from "../helpers/currency-format";
+import { useOrganization } from "../hooks/use-organization";
+import { BankAccount } from "../types/bank-account";
+import { TransactionsList } from "./transactions-list";
+import { datetime } from "../helpers/datetime-format";
+
+type Filters = {
+  status: BankAccount["status"];
+};
+
+type Filtered = {
+  total: number;
+  accounts: BankAccount[];
+};
+
+const IconAccountActive = {
+  source: Icon.Building,
+  tintColor: Color.PrimaryText,
+};
+
+const IconAccountClosed = {
+  source: Icon.Building,
+  tintColor: Color.Red,
+};
+
+export function AccountsList() {
+  const { push } = useNavigation();
+
+  const { organization } = useOrganization();
+
+  const [details, setDetails] = useState<boolean>(false);
+  const [filters, setFilters] = useState<Filters>({ status: "active" });
+
+  const filtered = useMemo<Filtered>(() => {
+    if (!organization)
+      return {
+        total: 0,
+        accounts: [],
+      };
+
+    const accounts = organization.bank_accounts.filter((account) => account.status === filters.status);
+    const total = accounts.reduce((total, account) => total + account.balance, 0);
+
+    return {
+      total,
+      accounts,
+    };
+  }, [organization, filters]);
+
+  return (
+    <List
+      isLoading={!organization}
+      isShowingDetail={details}
+      navigationTitle={currency.format(filtered.total)}
```

Comment:
```suggestion
```

Don't change the [navigationTitle](https://developers.raycast.com/basics/prepare-an-extension-for-store#navigation-title) in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context.

Reply:
Removed the `navigationTitle`